### PR TITLE
doc: How to update OCP version jobs in CI

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Fixes https://github.com/openshift/oadp-operator/issues/1229

### CODE REVIEW

**Comment by mateusoliveira43** [comment]
This PR is blocked by https://github.com/openshift/release/pull/45918

File: docs/developer/PROW_CI.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+## PROW CI
+
+The project uses [PROW CI](https://docs.ci.openshift.org/docs/) as a Continuous Integration/Delivery tool.
+
+The configuration for OADP repo can be found in
```
Comment (kaovilai) [inline docs/developer/PROW_CI.md:5]: perhaps add notes on plugins as well

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: docs/developer/PROW_CI.md (hunk 1 above)
Comment (kaovilai) [reply docs/developer/PROW_CI.md:5]: that's all

**Comment by weshayutin** [comment]
@mateusoliveira43 THANK YOU for taking the time to put this together.  Folks this is a very nice start, merging. Patch as required :)

**Pull request merged**